Feature/survey form integration

debadeep93 opened the pull request:
Integrated the survey forms to start with ati -> task -> resque
Important notes:

- task is now at uri:    /task

- requires worker id passed as 'wid' on base path for storing workerid

debadeep93 committed:
Added new html forms for surveys. Work in progress to integrate with flow

debadeep93 committed:
Removed unnessary form.

debadeep93 committed:
Integrated survey forms to flow.

debadeep93 committed:
Merge branch 'master' into feature/survey-form-integration

# Conflicts:
#	index.js
#	public/index.html

debadeep93 committed:
Made small changes to accomadate changed query param names

akshitgupta95 commented on public/forms/ati-ci.html:
```diff
@@ -0,0 +1,202 @@
+            </p>
+        </span>
+        <form id="survey-form" action="https://send.pageclip.co/ssJ875Dr8gsBFokkWaOxD7zvXK70aOkU/ati-ci-task" class="pageclip-form" method="post">
+            <input type="hidden" id="workerId" class="workerId" name="workerId" value="900">
```
when is this value being modified? wid will be in query paramter

akshitgupta95 replied:
what about scenario id? also, that needs to be stored

akshitgupta95 commented on public/forms/ati-ci.html:
```diff
@@ -0,0 +1,202 @@
+        var params = (new URL(document.location)).searchParams;
+        var workerID = params.get("wid");
+
+        console.log(workerID);
```
remove log

akshitgupta95 commented on public/forms/resque.html:
```diff
@@ -0,0 +1,485 @@
+                var params = (new URL(document.location)).searchParams;
+                var workerID = params.get("wid");
+
+            console.log(workerID);
```
remove log

akshitgupta95 commented on public/index.html:
```diff
@@ -54,6 +55,15 @@ <h2>Find your perfect house</h2>
+
+    $("#continue").click(function (event) {
+        window.location.href = url;
+        event.preventDefault();
```
move to seperate js file

akshitgupta95 commented on index.js:
```diff
@@ -73,45 +75,53 @@ const f_submit = require('./fulfillments/default/ConfirmSubmission');
-  intentMap.set('ConfirmSubmission', f_submit.fulfillment);
-  intentMap.set('tryagain', f_tryagain.fulfillment);
-
+app.get('/', function (req, res) {
```
this will cause conflict with express.static file serving. Move to some other path.

debadeep93 committed:
Changed endpoint of task to root.
Incorporated 'sid with RNG- random number generation between 1 and 3'.
Added endpoint for number of scenarios. Unimplemented rng with count fetch(running hard coded max value)

akshitgupta95 reviewed (approved):
Some changes are pending, rest is fine.
